Extension/Javascript-Extension/Visual-Studio-Code/code-snippet.md

```diff
@@ -34,20 +34,20 @@ The following steps help you to use the Syncfusion Angular code snippet in your
 
 5. After adding the Syncfusion Angular component to the html file, use the tab key to fill in the required values to render the component with data. You can also find the Syncfusion help link at the top of the added snippet to learn more about the new Syncfusion Angular component feature.
 
-    ![Help](images/Help.png)
+    ![Help in JavaScript Extension](images/javascript-extension-help.png)
```

Comment:
Change the alt text and image name like as below,

Help link added in JavaScript Code Snippet.

Reply:
Resolved as per the comment.